generate_instructions.py

```diff
@@ -81,28 +94,37 @@ def __init__(self, blocks, final_blocks=[]):
     def is_complete(self):
         return self.current_blocks == []
 
-    @staticmethod
-    def get_instruction(moved_block):
-        point = moved_block.position
-        phrase = 'move the %s %s block here' \
-                 % (moved_block.side1_color,
-                    moved_block.side1_letter)
-        return Instruction(phrase, point)
-
     def generate_action(self, goal_config):
```

Comment:
This method is getting a little large. Consider splitting some functionality out, such as creating the next configuration.